### ISSUE TITLE
Add TS 29.561 to wiki page about relevant 5G MBS specifications

### ISSUE THREAD

**Issue by TuanTranBPK** [5G-MAG/Standards#121]
I look around the documents and specifications on N6mb/Nmb9 for MB-UPF but don't find them on the wiki. It take a while to figure out the appropriate 3GPP specification.

**Comment by jordijoangimenez** [comment]
@rjb1000 @Borjis131 you may be across this. Could you have a quick look at the wiki https://github.com/5G-MAG/Standards/wiki/5G-Multicast-Broadcast-Services-(5MBS):-Relevant-Specifications and detect any missings?

There may be some interfaces between MBS core network components not referenced.

**Comment by rjb1000** [comment]
SA2's TS 23.247 defines reference points N6mb and Nmb9, and the NOTE to figure 5.1-2 specifies that they are functionally identical to each other.

- Clause 8.2 defines the user plane protocol stacks for shared MBS delivery at these reference points, including options for a UDP tunnel or just plain IP multicast.
- Clause 8.2 also specifies that the user plane protocol stack for individual delivery is that same as that specified in clause 8.3.1 of TS 23.501.

I searched in vain for a separate CT3 specification at this reference point. But maybe that's because the above is all that needs to be specified?

**Comment by Borjis131** [comment]
As @rjb1000 pointed out, Clause 8.2 of 3GPP TS 23.247 has the same information as Clause 20.3 of 3GPP TS 29.561

I do not know if there is more information about N6mb/N9mb in other 3GPP specifications but I suggest you to follow 3GPP TS 23.247 for the high level architecture and then extract the relevant bits of the protocols and APIs needed from the other specifications.

**Comment by Borjis131** [comment]
I completely agree with you @TuanTranBPK, starting with these topics always has a steep learning curve. The thing is, that adding another specification to the list that doesn't add more information can be a bit overwhelming for the user that is starting too.

Anyway, I am open to add it if you guys find it useful, this was just my opinion.

**Comment by TuanTranBPK** [comment]
The wiki page "5G-Multicast-Broadcast-Services-(5MBS):-RAN-Procedures-for-MBS-Broadcast" provides some good pointers to which spec/clause for a particular topic. It would be nice if we could do similar thing for the Core Network part, for example the pointers spec/clause for each interface related to 5G MBS.
